apps/admin/frontend/src/screens/reporting/single_precinct_tally_report_screen.tsx

```diff
@@ -32,27 +33,30 @@ export function SinglePrecinctTallyReportScreen(): JSX.Element {
   const [precinctId, setPrecinctId] = useState<string>();
 
   return (
-    <NavigationScreen title={TITLE} parentRoutes={reportParentRoutes}>
-      <SelectPrecinctContainer>
-        <P>Select Precinct:</P>
-        <SearchSelect
-          isMulti={false}
-          isSearchable
-          value={precinctId}
-          options={election.precincts.map((precinct) => ({
-            value: precinct.id,
-            label: precinct.name,
-          }))}
-          onChange={(value) => setPrecinctId(value)}
-          ariaLabel="Select Precinct"
+    <NavigationScreen title={TITLE} parentRoutes={reportParentRoutes} noPadding>
+      <ReportScreenContainer>
+        <SelectPrecinctContainer>
+          <span>Select Precinct:</span>
+          <SearchSelect
+            isMulti={false}
+            isSearchable
+            value={precinctId}
+            options={election.precincts.map((precinct) => ({
+              value: precinct.id,
+              label: precinct.name,
+            }))}
+            onChange={(value) => setPrecinctId(value)}
+            ariaLabel="Select Precinct"
+            style={{ width: '30rem' }}
```

Comment:
This change snuck in from my long election content polishing (long precinct names in this case)